app/src/sharedTest/java/org/oppia/android/app/topic/lessons/TopicLessonsFragmentTest.kt

```diff
@@ -281,8 +283,9 @@ class TopicLessonsFragmentTest {
       clickLessonTab()
       scrollToPosition(position = 1)
       clickStoryItem(position = 1, targetViewId = R.id.chapter_list_drop_down_icon)
-      onView(withId(R.id.story_summary_recycler_view)).perform(
-        actionOnItemAtPosition<RecyclerView.ViewHolder>(2, scrollTo())
+      recyclerViewScrollingActions.scrollToPosition(
+        R.id.story_summary_recycler_view,
+        2
```

Comment:
```suggestion
        position = 2
```

When using literals, prefer to add named arguments to provide context to readers on the value. Ditto elsewhere in this PR.

Reply:
Noted. Done.
Also created #2515 as a good-first-issue for such changes elsewhere in the codebase.

Reply:
This change will be handled in an independent PR, as planned, migration for positional use case will be handled independently for the entire codebase.